hermes/file_gen_test.go

```diff
@@ -0,0 +1,39 @@
+package hermes
+
+import(
+	//"fmt"
+	"testing"
+	"math/rand"
+	"strconv"
+	"time"
+	//"crypto/sha1"
+)
+
+
+func TestChecksum(t *testing.T){
+	var global_contents string = "jhfvjhdfjhfjjhjhdfvjvcvfjh"
+	var result_expected string = "_68f3caf439065824dcf75651c202e9f7c28ebf07"
+	file := HermesFile{}
+	file.contents = global_contents;
+	result := generate_file_checksum(file)
+	if result != result_expected {
+		t.Errorf("generate_file_checksum(\"jhfvjhdfjhfjjhjhdfvjvcvfjh\") failed expected %v got %v", result_expected, result)
```

Comment:
In case you want to change your file contents in the future, it's best to use the variable. It's also less human-error prone.

Ditto for the errors in TestFileName.
```suggestion
		t.Errorf("generate_file_checksum(\"%s\") failed expected %v got %v", file.contents, want, got)
```